### DISCUSSION TITLE
Is BIOS 313 (windows) as bad as it's said ?

### DISCUSSION THREAD

**Discussion by remiconnesson** [seerge/g-helper#1716] [General]
I recently removed armory crate as I was having a bad experience and replaced it with ghelpher (thank you 🙏)

Not immediately related to ghelper but I'm concerned about bios version 313. As people here seems to know what they are doing, I hope it's ok if I ask my question here.

My question is: Is BIOS 313 (windows) [as bad as it's said](https://www.reddit.com/r/ZephyrusG14/comments/x0jmc8/bios_313_a_survivors_guide/) ?

Currently I'm still having an issue with network connectivity (usually about 100mbs, for no reason it's sometimes drop to below 1mpbs, the only current fix I have found is [to reset network adapters, and reboot,](https://www.elevenforum.com/t/reset-network-adapters-in-windows-11.4513/) it fixes it for a time)

I also experience hard crashes when on google meet calls, or when I plug my laptop.

![image](https://github.com/seerge/g-helper/assets/28846642/a0e2f8f6-2306-4fb5-a409-e624a5a03c8f)

I don't know what I'm doing and I don't want to brick my computer.

What do you think about bios 313, should I update / revert ?

**Comment by seerge** [comment]
@remiconnesson hello, honestly i have no idea :) What you see here , is just a mirror of official asus website (all links go there). I personally always update bios to the latest
